docs/userdocs/k8s/README.md

```diff
@@ -22,19 +22,40 @@ Submarine for K8s supports distributed TensorFlow and PyTorch.
 
 Submarine can run on K8s >= 1.14, supports features like GPU isolation.
 
+We have validated Submarine on the following versions:
+
+[FIXME]: is it accurate?
+
+| K8s Version   | Support?  |
+| ------------- |:-------------:|
+| 1.13.x (or earlier) | X |
+| 1.14.x | √ |
+| 1.15.x | √ |
+| 1.16.x | √ |
+| 1.17.x | To be verified |
+| 1.17.x | To be verified |
+
 ## Install Submarine
-Submarine can be deployed on any K8s environment if version matches. If you don't have a running K8s, you can follow the steps to set up a K8s using [kind, Kubernetes-in-Docker](https://kind.sigs.k8s.io/) for testing purpose, we provides simple [tutorial](kind.md).
 
-### Use Helm Charts
+### Setup Kubernetes
+Submarine can be deployed on any K8s environment if version matches. If you don't have a running K8s, you can set up a K8s using [Docker Desktop](https://www.docker.com/products/docker-desktop), [MiniKube](https://kubernetes.io/docs/tasks/tools/install-minikube/), or [kind, Kubernetes-in-Docker](https://kind.sigs.k8s.io/).
+
+From our experiences, Docker Desktop is an easier choice.
+
+### Install Submarine Use Helm Charts
 After you have an up-and-running K8s, you can follow [Submarine Helm Charts Guide](helm.md) to deploy Submarine services on K8s cluster in minutes.
 
-## Use Submarine
+## Model Training (Experiment) on K8s
 
-### Model training (experiment) on K8s
+### Model Training With Submarine Python SDK
+
+Submarine Python SDK can runs on any machine and it will talk to Submarine Server via REST API. So you can install Submarine Python SDK on your laptop, a gateway machine, your favorite IDE (like PyCharm/Jupyter, etc.).
 
 #### Prepare Python Environment to run Submarine SDK
 
-Submarine SDK assumes Python3.7+ is ready.
+First of all
+
+Submarine SDK requires Python3.7+.
```

Comment:
```suggestion
Submarine SDK requires Python3.5+.
```
We support Python 3.5, 3.6, 3.7.
Tensorflow is not compatible with python3.8, so if they use deepfm or other predefined ctr models, they will get the error.